Tests/BuildTests/PrepareForIndexTests.swift

```diff
@@ -0,0 +1,62 @@
+// Copyright (C) 2024 Apple Inc. All rights reserved.
+//
+// This document is the property of Apple Inc.
+// It is considered confidential and proprietary.
+//
+// This document may not be reproduced or transmitted in any form,
+// in whole or in part, without the express written permission of
+// Apple Inc.
```

Comment:
This header is wrong, it should be replaced with the Apache 2.0 header.

Reply:
I also recommend running `./scripts/soundness.sh` before pushing commits, which runs the formatter and also checks if these license headers are valid.

Reply:
Thanks Max, yeah, I should probably remove that new file template too. :)